Integrations/integration-VirusTotal.yml

```diff
@@ -505,6 +513,89 @@ script:
         return {Type: entryTypes.note, Contents: res.body, ContentsFormat: formats.json, HumanReadable: res.obj.upload_url, EntryContext: {vtUploadURL: res.obj.upload_url}};
     }
 
+    var domainToIP = function(domain, threshold, longFormat, sampleSize, waitForRateLimit, retries) {
+        if (!sampleSize) {
+            sampleSize = 10;
+        }
+        if (!threshold) {
+            threshold = params.domainThreshold || 10;
+        }
+        threshold = parseInt(threshold)
+        var res = withRetries(waitForRateLimit, retries, function() {return doReq('GET', 'domain/report', {domain: domain});});
+        var o = res.obj;
+        var ec = {};
+        if (o.response_code === 0) {
+            ec.DBotScore = {Indicator: domain, Type: 'domain', Vendor: 'VirusTotal', Score: 0};
+            return {Type: entryTypes.note, Contents: res.body, ContentsFormat: formats.json, EntryContext: ec,
+                HumanReadable: 'VirusTotal does not have details about ' + domain + '\n' + res.obj.verbose_msg};
+        }
+
+        var md = '## VirusTotal result for Domain: ' + domain + '\n';
+        md += 'VT Link: [' + domain + '](https://www.virustotal.com/en/search?query=' + encodeURIComponent(domain) + ')\n';
+
+        if (o.resolutions && o.resolutions.length > 0) {
+            md += tableToMarkdown("IPs which resolve to this domain", o.resolutions, Object.keys(o.resolutions[0]), undefined, function (header) { return underscoreToCamelCase(header); });
+            if (ec.hasOwnProperty('Domain')){
+                ec.Domain.DNS = o.resolutions;
+            } else {
+                ec.Domain = {DNS: o.resolutions};
+            }
+        } else {
+            md += "No IPs found in the context of this domain\n";
+        }
+
+        return {Type: entryTypes.note, Contents: res.body, ContentsFormat: formats.json, HumanReadable: md, EntryContext: ec};
+    };
+
+    var ipToDomain = function(ip, longFormat, threshold, sampleSize, waitForRateLimit, retries) {
```

Comment:
@hirshbergshachar where is this func used?

Reply:
@meirwah old code leftovers, removed.